test/util/rational.cpp

```diff
@@ -19,14 +22,17 @@
 #include <boost/geometry/io/wkt/wkt.hpp>
 #include <boost/geometry/util/rational.hpp>
 
+using namespace boost::unit_test;
+
+BOOST_AUTO_TEST_SUITE(rational)
+
 void test_coordinate_cast(std::string const& s, int expected_nom, int expected_denom)
 {
-    boost::rational<int> a = bg::detail::coordinate_cast<boost::rational<int> >::apply(s);
+    boost::rational<int> a = bg::detail::coordinate_cast<boost::rational<int>>::apply(s);
```

Comment:
I am pretty sure `>>` will cause problems in with compilers compiling in C++03 mode.